src/framework/control.ml

```diff
@@ -260,7 +273,8 @@ struct
       WResult.init file; (* TODO: move this out of analyze_loop *)
 
     GU.global_initialization := true;
-    GU.earlyglobs := false;
+    (* GU.earlyglobs := false; *)
+    GU.earlyglobs := get_bool "exp.earlyglobs";
```

Comment:
If we already set it to `exp.earlyglobs` here, do we still need to set it again in Line 368?

Reply:
Nah, this just stayed around from when I was experimenting with this. That way it was easier to comment this first one out.